components/styled-links.js

```diff
@@ -15,9 +15,10 @@ const anchorStyles = {
 }
 
 const activeLinkStyles = (props, theme) => ({
```

Comment:
Wouldn't it be clearer to do  `const activeLinkStyles = props =>`?
Since this is how its done right now in `glamorous`.